### ISSUE TITLE
Bug: Adding new users to a team results in errors/odd behavior

### ISSUE THREAD

**Issue by angrygreenfrogs** [OneUptime/oneuptime#777]
**Describe the bug**
URL link in initial invitation email for a new user doesn't work, and attempting the forgot password process results in a TypeError server message

**To Reproduce**
1. Open Project Settings->View Team->Choose "Members"->Invite Team Member
2. I invited my gmail account angrygreenfrogs@gmail.com (never used in the system before this time)
3. Correctly received the "You have been invited to..." email
4. Chose to open the link in an incognito window to ensure I had a fresh session
5. Instead of logging me in or similar, all I saw was the standard Login screen and nothing else
6. I then thought to try the forgot password process
7. Received the forgot password email, clicked on that link to open it in the same incognito window
8. This resulted in this message: TypeError: Cannot read properties of null (reading 'toString')
![image](https://github.com/OneUptime/oneuptime/assets/413723/ef50bc72-4764-43e9-9251-8d279b9796f5)
9. However, oddly, it seemed to "work" to a degree because now when I opened oneuptime.com again, I was suddenly logged in

**Expected behavior**
Clearly the above included two bugs/incorrect behaviors, with both the initial link not working and with the TypeError issue

**Screenshots**
Attached above

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: Chrome
 - Version: 116.0.5845.188

**Comment by angrygreenfrogs** [comment]
Additional note:
I just tried this again and very simply inviting a new user does not work.

I invited another email address of mine to the team, and accepting the invitation does not work.

However, this time I noticed that when you click the accept link, a pop-up flashed by briefly saying "User should be logged in to access this API", then you get directed back to the login page.

I managed to get a quick screenshot as the popup shows for a second and then disappears
![image](https://github.com/OneUptime/oneuptime/assets/413723/14f7bf91-edf0-4e0a-ae29-304c3b3cbee1)
